frontend/pages/upload.tsx

```diff
@@ -1,3 +1,7 @@
+import Image from 'next/image';
+import { Inter } from 'next/font/google';
+import MatchMediaWrapper from '../components/MatchMediaWrapper';
+
```

Comment:
```suggestion
```

Unused/duplicates (really just revert all the changes to upload.tsx).

Reply:
Second reversion.  Out of scope of backend.